configs/SIRMIXALOT/config.h

```diff
@@ -26,6 +26,18 @@
 #define BOARD_NAME        SIRMIXALOT
 #define MANUFACTURER_ID   CUST
 
+
+#define USE_ACC
+#define USE_ACC_SPI_ICM20602
+#define USE_GYRO
+#define USE_GYRO_SPI_ICM20602
+#define USE_FLASH
+#define USE_FLASH_W25N01G
+#define USE_BARO
+#define USE_BARO_BMP280
+#define USE_BARO_SPI_BMP280
```

Comment:
There is no SPI define for baro, so we need I2C define
```suggestion
```

Reply:
sorry, pasted both and did not follow up.

Reply:
Updated the comment. You might have missed that :)